frappe_appointment/frappe_appointment/doctype/appointment_group/appointment_group_list.js

```diff
@@ -0,0 +1,12 @@
+frappe.listview_settings["Appointment Group"] = {
+  onload: function (listview) {
+    const user = frappe.session.user;
+    listview.page.add_button(__("My Appointments"), function () {
```

Comment:
`Filter by My Appointment Groups` is more intuitive 🤔 ?

Reply:
Won't that become a long button?

Reply:
What about `My Appointment Groups`?

Reply:
`My Appointment Groups` - The button action is unclear for this name. It could be a dialog, a dropdown, or something else. @shivamsn97 🤔

Reply:
@KanchanChauhan please let me know what should be the name for this button.

Reply:
@zeel-codder Updated to `Filter My Groups` as suggested by Kanchan.